Music app/Classes/PresentationLayer/Screens/Main/HeaderTableViewCell/HeaderTableViewCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  HeaderTableViewCell.swift
+//  Music app
+//
+//  Created by Tsaruk Nick on 6.08.23.
+//
+
+import UIKit
+
+protocol HeaderTableViewCellDelegate: AnyObject {
+    func goToUserProfile()
+    func cleanKeychain()
+}
+
+
```

Comment:
?

Reply:
Did not find it but this file is right now